test/integration/targets/eos_facts/tests/cli/default_facts.yaml

```diff
@@ -23,7 +23,7 @@
       # Items from those subsets are present
       - "result.ansible_facts.ansible_net_filesystems is defined" #hw
       - "result.ansible_facts.ansible_net_memtotal_mb > 10" #hw
-      - "result.ansible_facts.ansible_net_interfaces.Management1.ipv4.masklen > 1" # interfaces
+      - "result.ansible_facts.ansible_net_interfaces.Management1" # interfaces
```

Comment:
Is there anything we _can_ assert about this?

Reply:
We can, but we should update the testing here to create the ip address first.